backend/controllers/alert.js

```diff
@@ -0,0 +1,112 @@
+const moment = require('moment')
+const { formatTimestampToDate, properCaseName } = require('../utils')
+const config = require('../config')
+const { logError } = require('../logError')
+
+const serviceUnavailableMessage = 'Sorry, the service is unavailable'
+const getOffenderUrl = offenderNo => `${config.app.notmEndpointUrl}offenders/${offenderNo}`
+
+const alertFactory = (oauthApi, elite2Api) => {
+  const renderTemplate = (req, res, pageData) => {
+    const { alert, pageErrors, offenderDetails, ...rest } = pageData
+    const formAction =
+      offenderDetails && alert ? `/api/close-alert/${offenderDetails.bookingId}/${alert.alertId}` : undefined
```

Comment:
You could probably just use` offenderDetails && alert &&  `/api/close-alert/${offenderDetails.bookingId}/${alert.alertId}``